### PR TITLE
Update translation of korean

### PR DESCRIPTION
**Pull request by joonas-yoon**
**in sync with `main` branch**
- fixes typos on not translated parts
- added new links
- removed some sections (OOP, System routine)

Fix wrong links on indexes and translate more sentences in korean.

### CODE REVIEW

**Comment by joonas-yoon** [comment]
related on #118 for refer

**Comment by jwasham** [comment]
Thank you! Would you like me to pull this change now?

**Comment by joonas-yoon** [comment]
sure.

File: README.md [hunk 1]
```diff
@@ -830,7 +830,7 @@ Write code on a whiteboard or paper, not a computer. Test with some sample input
         - [ ] is_empty() - returns true if heap contains no elements
         - [ ] extract_max - returns the max item, removing it
         - [ ] sift_down - needed for extract_max
-        - [ ] remove(i) - removes item at index x
+        - [ ] remove(x) - removes item at index x
```
Comment (joonas-yoon) [inline README.md:836]: @jwasham ah, this has to be noted.

I did not mean to contain that commit, but it is useful.
It is just typo, should be make an issue for it?

and in my opinion, this looks like more accurate:
```
- - [ ] remove(x) - removes item at index x
+ - [ ] remove(i) - removes item at index i
```

the `i` for index, not the `x`, what do you think about this?

Comment (jwasham) [reply README.md:836]: Making them match is logical. Feel free to open an issue.

**Pull request merged**